src/main/scala/com/ing/wbaa/rokku/proxy/api/PostRequestActions.scala

```diff
@@ -43,7 +44,10 @@ trait PostRequestActions {
   private[this] def createBucketNotification(response: HttpResponse, httpRequest: HttpRequest, s3Request: S3Request,
       userSTS: User)(implicit id: RequestId): Future[Done] =
     httpRequest.method match {
-      case HttpMethods.POST | HttpMethods.PUT | HttpMethods.DELETE if bucketNotificationEnabled && (response.status == StatusCodes.OK || response.status == StatusCodes.NoContent) =>
+      case HttpMethods.POST | HttpMethods.PUT if bucketNotificationEnabled && (response.status == StatusCodes.OK || response.status == StatusCodes.NoContent) =>
+        MetricsFactory.incrementObjectsUploaded(httpRequest.method.name)
+        emitEvent(s3Request, httpRequest.method, userSTS.userName.value, awsRequestFromRequest(httpRequest))
+      case HttpMethods.DELETE if bucketNotificationEnabled && (response.status == StatusCodes.OK || response.status == StatusCodes.NoContent) =>
```

Comment:
This change doesn't alter any functionality right? Why did we separate the DELETE?

Reply:
I combined it back, I had initially different idea...